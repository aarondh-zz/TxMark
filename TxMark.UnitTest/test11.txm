<div>
  (a:
    "one",
    "two",
    "three",
    "four",
    "five",
    "sex"
  )
  (set: $array to (a:"one","two","three","four","five","sex"))
  (each: $array as $item)[
    Entry: $item
  ]
</div>
