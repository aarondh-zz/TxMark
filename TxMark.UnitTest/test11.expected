<div>
one, two, three, four, five and sex


Entry: one

Entry: two

Entry: three

Entry: four

Entry: five

Entry: sex

</div>
